test/style.js

```diff
@@ -118,19 +118,20 @@ describe('style', function() {
   });
   it('applies tranform as transition on remove', function(done) {
     var btn = h('button', { style: {
-        transition: 'transform 0.5s',
+        transition: 'transform 0.1s',
```

Comment:
Shortening the animation was my suggestion. For the sake of a faster test suite.

Reply:
That's a good idea :+1: